src/posts/asking-why-a-practice-is-not-able-to-accommodate-you.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Asking why a practice is not able to accommodate you
+date: '2021-01-10'
+tags:
+  - ada
+  - oral-deaf
+  - deaf
+  - telemedicine
+---
+
+# Context
+
+## Who was this written by?
+
+This text was written by Mel Chua.
```

Comment:
Do you think we should add an author field in the frontmatter YAML, like `author: Mel Chua`? I'm happy to code this in or help code it in. In the absence of that, this is good!